internal/k8shandler/clusterrestart.go

```diff
@@ -409,8 +409,9 @@ func (r *Restarter) setClusterConditions(updateStatus func()) {
 
 	r.mainCondition = func() bool {
 		return containsClusterCondition(api.Restarting, v1.ConditionTrue, r.clusterStatus) &&
-			containsClusterCondition(api.UpdatingESSettings, v1.ConditionFalse, r.clusterStatus) &&
-			containsClusterCondition(api.Recovering, v1.ConditionFalse, r.clusterStatus)
+			containsClusterCondition(api.UpdatingESSettings, v1.ConditionFalse, r.clusterStatus)
+		// &&
+		// containsClusterCondition(api.Recovering, v1.ConditionFalse, r.clusterStatus)
```

Comment:
i think we should revert the changes in this file... i'm thinking that we actually want to blank things out in the secret controller in the case where we are already mid restart (so we can force it to restart the cert restart process)